Playbooks/playbook-Phishing_Investigation_-_Generic_v2.yml

```
@@ -122,128 +106,115 @@ tasks:
-        "position": {
-          "x": 7.5,
-          "y": 195
-        }
-      }
+    view: "{\n  \"position\": {\n    \"x\": 7.5,\n    \"y\": 195\n  }\n}"
     note: false
     timertriggers: []
     ignoreworker: false
-  "12":
-    id: "12"
-    taskid: 0aa7f6c4-71f4-4753-8512-dd3a9d2cc9c7
+  '12':
+    id: '12'
+    taskid: 3fa6bd94-e6fa-4cad-8e42-a35aa3370a56
     type: regular
     task:
-      id: 0aa7f6c4-71f4-4753-8512-dd3a9d2cc9c7
+      id: 3fa6bd94-e6fa-4cad-8e42-a35aa3370a56
       version: -1
       name: Store the email address of the reporting user
       description: Store the email address of the user that reported the incident.
       scriptName: Set
       type: regular
       iscommand: false
-      brand: ""
+      brand: ''
     nexttasks:
       '#none#':
-      - "53"
+      - '53'
+      - '85'
     scriptarguments:
-      append:
-        simple: "true"
+      append: {}
       key:
-        simple: Account.Email.Address
+        simple: ReporterAddress
       value:
         complex:
           root: incident
           accessor: labels.Email/from
     reputationcalc: 1
     separatecontext: false
-    view: |-
-      {
-        "position": {
-          "x": 1182.5,
-          "y": 345
-        }
-      }
+    view: "{\n  \"position\": {\n    \"x\": 1182.5,\n    \"y\": 350\n  }\n}"
     note: false
     timertriggers: []
     ignoreworker: false
-  "13":
-    id: "13"
-    taskid: 86dd46bf-ba4d-4393-8fb6-b9da6e562b2e
+  '13':
+    id: '13'
+    taskid: 9551bc3f-272d-4db9-8a40-30b48d1ba33b
     type: regular
     task:
-      id: 86dd46bf-ba4d-4393-8fb6-b9da6e562b2e
+      id: 9551bc3f-272d-4db9-8a40-30b48d1ba33b
       version: -1
       name: Acknowledge incident was received
-      description: |
-        Send an auto-response to user that reported the incident, informing them the incident was received and being handled.
+      description: 'Send an auto-response to user that reported the incident, informing
+        them the incident was received and being handled.
+
+        '
       script: '|||send-mail'
       type: regular
       iscommand: true
-      brand: ""
+      brand: ''
     nexttasks:
       '#none#':
-      - "2"
+      - '2'
     scriptarguments:
+      additionalHeader: {}
+      attachCIDs: {}
       attachIDs: {}
       attachNames: {}
       bcc: {}
       body:
-        simple: "Hi ${.=val.Account.DisplayName && val.Email.Address === val.incident.labels['Email/from']
-          ? val.Account.DisplayName : val.incident.labels['Email/from']},\nWe've received
-          your email and are investigating.\nPlease do not touch the email until further
-          notice.\n\nCordially, \n  Your friendly neighborhood security team"
+        simple: "Hi ${.=function(val) { var reporter = val.ReporterAddress ;var account\
+          \ = val.Account.filter(function(acc) { return acc.DisplayName && acc.Email\
+          \ && acc.Email.Address === reporter }); return account[0] && account[0].DisplayName\
```

```suggestion
          \ && acc.Email.Address === reporter }); return account && account[0] && account[0].DisplayName\
```
just to be sure, if context has no `Account`